appservice/src/utils/azureClients.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApplicationInsightsManagementClient } from '@azure/arm-appinsights';
+import { WebSiteManagementClient } from '@azure/arm-appservice';
+import { ResourceManagementClient } from '@azure/arm-resources';
+import { createAzureClient, ISubscriptionContext } from 'vscode-azureextensionui';
+
+// Lazy-load @azure packages to improve startup performance.
+// NOTE: The client is the only import that matters, the rest of the types disappear when compiled to JavaScript
+
+export async function createWebSiteClient<T extends ISubscriptionContext>(context: T): Promise<WebSiteManagementClient> {
+    return createAzureClient(context, (await import('@azure/arm-appservice')).WebSiteManagementClient);
```

Comment:
Once it's imported, if we run createWebSiteClient again, the await should return immediately, right?

Reply:
Yes - Brandon even did some testing to verify

https://github.com/microsoft/vscode-docker/pull/2368#discussion_r498988999